core/Pos/Core/Constants.hs

```diff
@@ -80,23 +80,23 @@ isDevelopment = True
 isDevelopment = False
 #endif
 
--- | System start time embeded into binary.
-staticSysStart :: Maybe Timestamp
+-- | System start time embedded into binary.
+staticSysStart :: Timestamp
 staticSysStart
-    | isDevelopment = Nothing
-    | st > 0        = Just $ Timestamp $ sec st
+    | isDevelopment    = error "System start time should be passed \
+                              \as a command line argument in dev mode."
+    | st >= 1400000000 = Timestamp $ sec st
```

Comment:
Just some time that we certainly would not choose as start time 'cause it's far away enough from now